var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -138,6 +141,7 @@ class Libfabric(AutotoolsPackage, CudaPackage):
 
     conflicts("@1.9.0", when="platform=darwin", msg="This distribution is missing critical files")
     conflicts("fabrics=opx", when="@:1.14.99")
+    conflicts("fabrics=lnx", when="@:1")
```

Comment:
I would break out the lnx addition and the version updates into separate commits.

Reply:
Why, this gets squashed anyway?

Reply:
Yeah, forget we squash merge for Spack. I tend to prefer atomic commits for easy bisect/revert to debug issues.